src/components/Calculator.js

```diff
@@ -1,77 +1,67 @@
-import React from 'react';
+import { useState } from 'react';
 import calculate from '../logic/calculate';
+import './Calculator.css';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    // Ensure that props can be accessed in order to inherit parent's constuctor method.
-    super(props);
+// Refactoring class component to function component which uses a React hook
+const Calculator = () => {
+  // initializing current state `initCalc` and function that updates the state `setCalculater`
+  const [initCalc, setCalculator] = useState({ total: null, next: null, operation: null });
 
-    // Setting up initial states to null
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-
-    // Binding in order to `this` work even in the callback
-    this.handleClick = this.handleClick.bind(this);
-  }
-
-  // Defining function to handle click events
-  handleClick = (event) => {
-    // Changes the state to the result of the `calculate` function
-    this.setState((state) => calculate(state, event.target.innerHTML));
-  }
-
-  render() {
-    // To destructure this.state to individual variables
-    const { total, next, operation } = this.state;
+  // Defining function to handle click events and update state
+  const handleClick = (event) => {
+    // Changes initial calculator state by calling imported `calculate` function
+    setCalculator(calculate(initCalc, event.target.innerHTML));
+  };
+  // Definfing HTML that should be rendered to the DOM
+  return (
+    <div className="container-fluid d-md-flex align-items-start justify-content-center">
+      <div className="calc-text px-2 pt-5">
```

Comment:
- [ ] [OPTIONAl] well done creating the application :clap: how about decreasing the max-width for the calculator, currently it is not looking as a calculator and is taking a lot of the space, i think giving an appropriate max-width will improve the look and feel of the calculator please see :point_down: 

![image](https://user-images.githubusercontent.com/97521557/213419643-4c2493f5-16ac-454d-a452-f2428771d602.png)